General Refactor and Documentation Updates (#287)
DFE-Digital/find-a-lost-trn: README.md

bin/dev
```

### Basic Auth

> Due to the `Service open` feature flag which is set to `false` by default across all environments except Production, you will be asked to sign in with basic auth. You can find credentials in the .env._environment_ file stored in the ENV variables `SUPPORT_USERNAME` and `SUPPORT_PASSWORD`. You can toggle the `Service open` feature flag to `true` when developing locally to spare you having to always use basic auth.

Reviewer: ```suggestion
Due to the `Service open` feature flag which is set to `false` by default across all environments except Production, you will be asked to sign in with basic auth. You can find credentials in the .env._environment_ file stored in the ENV variables `SUPPORT_USERNAME` and `SUPPORT_PASSWORD`. You can toggle the `Service open` feature flag to `true` when developing locally to switch off basic auth.
```

Author: Suggestion updated

When you send an email locally, the email should appear in the message log in
the Notify dashboard in the `API integration` section.

> _Note:_ It is acceptable to set `GOVUK_NOTIFY_API_KEY=fake-key` when trying to quickly run the project locally

Reviewer: ```suggestion
_Note:_ You can set `GOVUK_NOTIFY_API_KEY=fake-key` when running locally if you don't need to use Notify.
```

Author: Suggestion updated

To run the tests (requires Chrome due to
[cuprite](https://github.com/rubycdp/cuprite)):

> Run the command below to allow assets to be compiled and cached up front as tests that utilise cuprite will need this

Reviewer: ```suggestion
To compile assets up front (needed by the end to end tests):
```

```bash
bin/rails assets:precompile
```

> And then

Reviewer: ```suggestion
And then:
```

Author: Suggestion updated
